sdk/helper/pluginutil/run_config.go

```diff
@@ -139,7 +143,8 @@ func (rc runConfig) makeConfig(ctx context.Context) (*plugin.ClientConfig, error
 		clientConfig.SkipHostEnv = true
 		clientConfig.RunnerFunc = containerCfg.NewContainerRunner
 		clientConfig.UnixSocketConfig = &plugin.UnixSocketConfig{
-			Group: strconv.Itoa(containerCfg.GroupAdd),
+			Group:   strconv.Itoa(containerCfg.GroupAdd),
+			TempDir: os.Getenv("VAULT_PLUGIN_TMPDIR"),
```

Comment:
Maybe this should be settable with both an env var and a config file option?

Reply:
It would be nice, but not strictly necessary I think.

Reply:
👍 I think I'll aim to add a config option for 1.15.1 given where we are in the release cycle.